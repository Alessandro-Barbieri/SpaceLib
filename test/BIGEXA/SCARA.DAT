3

0
0 0 0 0.33 0
10
0.03 0 0
  0.03 0
	0.05
-0.05 0 0

0
0 0 0 0.33 0
10
0.03 0 0
  0.03 0
	0.05
-0.05 0 0

1
0 -0.1 0 0 3.14159
3
0.008 0  0
   0.008 0
	0.0015
0 0 -0.10

0 0 -9.8
0 0 0 0 0 0