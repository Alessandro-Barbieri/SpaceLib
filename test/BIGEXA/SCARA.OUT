

 Link 1


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.3300
  0.0000	  1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  1.0000	  0.0000	  0.0000	  0.3300
  0.0000	  1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -1.0000	  0.0000	  0.0000
  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -1.0000	  0.0000	  0.0000
  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -1.0000	  0.0000	  0.0000
  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -1.0000	 -10.0000	  0.0000	  0.0000
 10.0000	 -1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -1.0000	 -10.0000	  0.0000	  0.0000
 10.0000	 -1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -1.0000	 -10.0000	  0.0000	  0.0000
 10.0000	 -1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  0.0500	  0.0000	  0.0000	 -0.5000
  0.0000	  0.0250	  0.0000	  0.0000
  0.0000	  0.0000	  0.0050	  0.0000
 -0.5000	  0.0000	  0.0000	 10.0000

 Inertia Matrix in frame (0)                O

  0.8090	  0.0000	  0.0000	  2.8000
  0.0000	  0.0250	  0.0000	  0.0000
  0.0000	  0.0000	  0.0050	  0.0000
  2.8000	  0.0000	  0.0000	 10.0000

 Total actions                             FI

  0.0000	  8.3400	 27.4400	  2.8000
 -8.3400	  0.0000	  0.0000	 -28.0000
 -27.4400	  0.0000	  0.0000	 -98.0000
 -2.8000	 28.0000	 98.0000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 107.3910	 106.6240	 41.2000
 -107.3910	  0.0000	  0.0000	 -184.6000
 -106.6240	 -0.0000	  0.0000	 -225.4000
 -41.2000	 184.6000	 225.4000	  0.0000


 Link 2


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.3300
  0.0000	  1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  1.0000	  0.0000	  0.0000	  0.6600
  0.0000	  1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -2.0000	  0.0000	  0.0000
  2.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -2.0000	  0.0000	  0.0000
  2.0000	  0.0000	  0.0000	 -0.6600
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -3.0000	  0.0000	  0.0000
  3.0000	  0.0000	  0.0000	 -0.6600
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -4.0000	 -20.0000	  0.0000	  0.0000
 20.0000	 -4.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -4.0000	 -20.0000	  0.0000	  1.3200
 20.0000	 -4.0000	  0.0000	 -6.6000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -9.0000	 -30.0000	  0.0000	  2.6400
 30.0000	 -9.0000	  0.0000	 -6.6000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  0.0500	  0.0000	  0.0000	 -0.5000
  0.0000	  0.0250	  0.0000	  0.0000
  0.0000	  0.0000	  0.0050	  0.0000
 -0.5000	  0.0000	  0.0000	 10.0000

 Inertia Matrix in frame (0)                O

  3.7460	  0.0000	  0.0000	  6.1000
  0.0000	  0.0250	  0.0000	  0.0000
  0.0000	  0.0000	  0.0050	  0.0000
  6.1000	  0.0000	  0.0000	 10.0000

 Total actions                             FI

  0.0000	 72.8700	 59.7800	 28.5000
 -72.8700	  0.0000	  0.0000	 -117.0000
 -59.7800	  0.0000	  0.0000	 -98.0000
 -28.5000	 117.0000	 98.0000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 99.0510	 79.1840	 38.4000
 -99.0510	  0.0000	  0.0000	 -156.6000
 -79.1840	 -0.0000	  0.0000	 -127.4000
 -38.4000	 156.6000	 127.4000	  0.0000


 Link 3


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	 -1.0000	 -0.0000	  0.0000
  0.0000	  0.0000	 -1.0000	 -0.1000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  1.0000	  0.0000	  0.0000	  0.6600
  0.0000	 -1.0000	 -0.0000	  0.0000
  0.0000	  0.0000	 -1.0000	 -0.1000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -3.0000	  0.0000	  0.0000
  3.0000	  0.0000	  0.0000	 -0.6600
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -9.0000	 -30.0000	  0.0000	  2.6400
 30.0000	 -9.0000	  0.0000	 -6.6000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  0.0007	  0.0000	  0.0000	  0.0000
  0.0000	  0.0008	  0.0000	  0.0000
  0.0000	  0.0000	  0.0373	 -0.3000
  0.0000	  0.0000	 -0.3000	  3.0000

 Inertia Matrix in frame (0)                O

  1.3076	  0.0000	 -0.0000	  1.9800
  0.0000	  0.0008	  0.0000	  0.0000
 -0.0000	  0.0000	  0.0072	 -0.0000
  1.9800	  0.0000	 -0.0000	  3.0000

 Total actions                             FI

  0.0000	 26.1810	 19.4040	  9.9000
 -26.1810	  0.0000	  0.0000	 -39.6000
 -19.4040	 -0.0000	  0.0000	 -29.4000
 -9.9000	 39.6000	 29.4000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 26.1810	 19.4040	  9.9000
 -26.1810	  0.0000	  0.0000	 -39.6000
 -19.4040	 -0.0000	  0.0000	 -29.4000
 -9.9000	 39.6000	 29.4000	  0.0000


 Link 1


 Relative Position Matrix                   A

  0.9950	 -0.0998	  0.0000	  0.3284
  0.0998	  0.9950	  0.0000	  0.0329
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  0.9950	 -0.0998	  0.0000	  0.3284
  0.0998	  0.9950	  0.0000	  0.0329
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -1.1000	  0.0000	  0.0000
  1.1000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -1.1000	  0.0000	  0.0000
  1.1000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -1.1000	  0.0000	  0.0000
  1.1000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -1.2100	 -11.0000	  0.0000	  0.0000
 11.0000	 -1.2100	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -1.2100	 -11.0000	  0.0000	  0.0000
 11.0000	 -1.2100	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -1.2100	 -11.0000	  0.0000	  0.0000
 11.0000	 -1.2100	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  0.0500	  0.0000	  0.0000	 -0.5000
  0.0000	  0.0250	  0.0000	  0.0000
  0.0000	  0.0000	  0.0050	  0.0000
 -0.5000	  0.0000	  0.0000	 10.0000

 Inertia Matrix in frame (0)                O

  0.8012	  0.0779	  0.0000	  2.7860
  0.0779	  0.0328	  0.0000	  0.2795
  0.0000	  0.0000	  0.0050	  0.0000
  2.7860	  0.2795	  0.0000	 10.0000

 Total actions                             FI

  0.0000	  9.1740	 27.3029	  6.4459
 -9.1740	  0.0000	  2.7394	 -30.3079
 -27.3029	 -2.7394	  0.0000	 -98.0000
 -6.4459	 30.3079	 98.0000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 114.3210	 105.5365	 78.4527
 -114.3210	  0.0000	 14.3156	 -187.8961
 -105.5365	 -14.3156	  0.0000	 -225.4000
 -78.4527	 187.8961	 225.4000	  0.0000


 Link 2


 Relative Position Matrix                   A

  0.9950	 -0.0998	  0.0000	  0.3284
  0.0998	  0.9950	  0.0000	  0.0329
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  0.9801	 -0.1987	  0.0000	  0.6518
  0.1987	  0.9801	  0.0000	  0.0985
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -2.1000	  0.0000	  0.0000
  2.1000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -2.1000	  0.0000	  0.0692
  2.1000	  0.0000	  0.0000	 -0.6895
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -3.2000	  0.0000	  0.0692
  3.2000	  0.0000	  0.0000	 -0.6895
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -4.4100	 -21.0000	  0.0000	  0.0000
 21.0000	 -4.4100	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -4.4100	 -21.0000	  0.0000	  2.1399
 21.0000	 -4.4100	  0.0000	 -6.7501
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -10.2400	 -32.0000	  0.0000	  3.6569
 32.0000	 -10.2400	  0.0000	 -6.5979
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  0.0500	  0.0000	  0.0000	 -0.5000
  0.0000	  0.0250	  0.0000	  0.0000
  0.0000	  0.0000	  0.0050	  0.0000
 -0.5000	  0.0000	  0.0000	 10.0000

 Inertia Matrix in frame (0)                O

  3.6583	  0.5339	  0.0000	  6.0277
  0.5339	  0.1035	  0.0000	  0.8857
  0.0000	  0.0000	  0.0050	  0.0000
  6.0277	  0.8857	  0.0000	 10.0000

 Total actions                             FI

  0.0000	 77.3675	 59.0715	 53.4982
 -77.3675	  0.0000	  8.6801	 -117.8377
 -59.0715	 -8.6801	  0.0000	 -98.0000
 -53.4982	 117.8377	 98.0000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 105.1470	 78.2336	 72.0067
 -105.1470	  0.0000	 11.5762	 -157.5882
 -78.2336	 -11.5762	  0.0000	 -127.4000
 -72.0067	 157.5882	 127.4000	  0.0000


 Link 3


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	 -1.0000	 -0.0000	  0.0000
  0.0000	  0.0000	 -1.0000	 -0.1000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  0.9801	  0.1987	  0.0000	  0.6518
  0.1987	 -0.9801	 -0.0000	  0.0985
  0.0000	  0.0000	 -1.0000	 -0.1000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -3.2000	  0.0000	  0.0692
  3.2000	  0.0000	  0.0000	 -0.6895
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -10.2400	 -32.0000	  0.0000	  3.6569
 32.0000	 -10.2400	  0.0000	 -6.5979
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  0.0007	  0.0000	  0.0000	  0.0000
  0.0000	  0.0008	  0.0000	  0.0000
  0.0000	  0.0000	  0.0373	 -0.3000
  0.0000	  0.0000	 -0.3000	  3.0000

 Inertia Matrix in frame (0)                O

  1.2752	  0.1926	 -0.0000	  1.9553
  0.1926	  0.0299	  0.0000	  0.2955
 -0.0000	  0.0000	  0.0072	 -0.0000
  1.9553	  0.2955	 -0.0000	  3.0000

 Total actions                             FI

  0.0000	 27.7795	 19.1621	 18.5085
 -27.7795	  0.0000	  2.8961	 -39.7505
 -19.1621	 -2.8961	  0.0000	 -29.4000
 -18.5085	 39.7505	 29.4000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 27.7795	 19.1621	 18.5085
 -27.7795	  0.0000	  2.8961	 -39.7505
 -19.1621	 -2.8961	  0.0000	 -29.4000
 -18.5085	 39.7505	 29.4000	  0.0000